modules/lvms-unsupported-devices.adoc

```diff
@@ -6,9 +6,9 @@
 [id="lvms-unsupported-devices_{context}"]
 = Devices not supported by {lvms}
 
-When you are adding the paths to the devices in the `deviceSelector` section of the `LVMCluster` YAML, ensure that the devices are supported by {lvms}. If you add paths to the unsupported devices, then {lvms} excludes the devices to avoid complexity in managing logical volumes.
+When you are adding the device paths in the `deviceSelector` field of the `LVMCluster` cusotm resource (CR), ensure that the devices are supported by {lvms}. If you add paths to the unsupported devices, {lvms} excludes the devices to avoid complexity in managing logical volumes.
```

Comment:
```suggestion
When you are adding the device paths in the `deviceSelector` field of the `LVMCluster` custom resource (CR), ensure that the devices are supported by {lvms}. If you add paths to the unsupported devices, {lvms} excludes the devices to avoid complexity in managing logical volumes.
```

Reply:
Thank you for the review, @xenolinux!
Fixed the typo.